examples/k8s.yaml

```diff
@@ -141,17 +141,17 @@ spec:
         prometheus.io/scrape: "true"
       labels:
         app: opentelemetry
-        component: otelsvc-collector
+        component: otel-collector
     spec:
       containers:
       - command:
           - "/otelsvc_collector_linux"
-          - "--config=/conf/otelsvc-collector-config.yaml"
+          - "--config=/conf/otel-collector-config.yaml"
         env:
         - name: GOGC
           value: "80"
         image: omnition/opencensus-collector:0.1.6
```

Comment:
not from your change but this is incorrect, we need to review this whole file.

Reply:
Thanks @ccaraman https://github.com/open-telemetry/opentelemetry-service/issues/366